windows-agent/cmd/ubuntu-pro-agent/main.go

```diff
@@ -66,6 +66,16 @@ func setLoggerOutput(a app) (func(), error) {
 
 	logFile := filepath.Join(publicDir, "log")
 
+	// Move old log file
+	fileInfo, err := os.Stat(logFile)
+	if err == nil && fileInfo.Size() > 0 {
+		oldLogFile := filepath.Join(publicDir, "log.old")
+		err = os.Rename(logFile, oldLogFile)
+		if err != nil {
+			log.Warnf("Could not archive previous log file: %v", err)
```

Comment:
As you can see this line is not covered in tests. I think we can cover that one (even on Windows) by creating a folder instead of a file. Ping me once you are around and we can have pair on that together.